Makefile

```diff
@@ -9,20 +9,21 @@ IMAGE_TAG           := $(shell cat VERSION)
 PROVIDER_NAME       := Azure
 PROJECT_NAME        := gardener
 CONTROL_NAMESPACE  := default
-CONTROL_KUBECONFIG := dev/control-kubeconfig.yaml
-TARGET_KUBECONFIG  := dev/target-kubeconfig.yaml
+CONTROL_KUBECONFIG := ./dev/control-kubeconfig.yaml
+TARGET_KUBECONFIG  := ./dev/target-kubeconfig.yaml
```

Comment:
why are these changes required?

Reply:
They are not needed, I noticed it after I made the commit, so I let it be. I'll revert it back.